script/tool/lib/src/podspec_check_command.dart

```diff
@@ -110,6 +110,11 @@ class PodspecCheckCommand extends PackageLoopingCommand {
   }
 
   Future<List<File>> _podspecsToLint(RepositoryPackage package) async {
+    // Since the pigeon podspecs require generated files that are not included in git,
+    // the podspec lint fails.
+    if (package.displayName == 'packages/pigeon') {
```

Comment:
It's only the platform_tests, right? I would prefer to make this more targeted, filtering out any results that contain `packages/pigeon/platform_tests/` in the output, because the more specific we make anything that skips CI steps, the less chance of us accidentally having an unintended skip that we never notice (until it bites us).